src/Controllers/UserController.php

```diff
@@ -35,4 +38,19 @@ public function show( Request $request, Response $response, $id ) {
 			'user' => $user
 		] );
 	}
+
+	public function login( Request $request, Response $response){
+	
+		$require_login = get_option('require_login');
+
+		if ($require_login === 1) {
```

Comment:
@Perumal605 , please the require login check here since this is now handled by the LoggedOut middleware.